proto/axelar/axelarnet/v1beta1/query.proto

```diff
@@ -21,3 +21,39 @@ message PendingIBCTransferCountResponse {
 message ParamsRequest {}
 
 message ParamsResponse { Params params = 1 [ (gogoproto.nullable) = false ]; }
+
+enum ChainStatus {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  CHAIN_STATUS_UNSPECIFIED = 0
+      [ (gogoproto.enumvalue_customname) = "StatusUnspecified" ];
+  CHAIN_STATUS_ACTIVATED = 1 [ (gogoproto.enumvalue_customname) = "Activated" ];
+  CHAIN_STATUS_DEACTIVATED = 2
+      [ (gogoproto.enumvalue_customname) = "Deactivated" ];
+}
+
+// ChainsRequest represents a message that queries all registered cosmos chains
+// by status
+message ChainsRequest { ChainStatus status = 1; }
```

Comment:
I prefer having this query in the nexus module, i.e. GetCosmoaChains and GetEVMChains or GetChains where you can specify the type.

Reply:
good point. I do like to have wrapper queries so you don't have to specify the module type to query the chains. But it makes sense to refactor and reuse the same query underthehood.

Reply:
@cgorenflo thoughts? we'll do a refactor?

Reply:
yeah the chains query should go into the nexus, otherwise you'll return evm chains in the axelarnet module, which is very weird

Reply:
refactored chains query in #2117